.github/workflows/publish-testpypi.yml

```diff
@@ -21,29 +21,55 @@ jobs:
       - name: Install Python dependencies
         run: |
           python -m pip install --upgrade pip
-          make install requirements
+          make install
           pip install setuptools wheel twine
-      - name: Build and publish
-        env:
-          TWINE_USERNAME: ${{ secrets.TESTPYPI_USERNAME }}
-          TWINE_PASSWORD: ${{ secrets.TESTPYPI_PASSWORD }}
+      - name: Build mitiq
+        run: python setup.py sdist bdist_wheel
```

Comment:
This way of building the wheel is deprecated, see table at the bottom of https://blog.ganssle.io/articles/2021/10/setup-py-deprecated.html#summary. The new way is
```
pip install build
python -m build
```

Reply:
Done in https://github.com/unitaryfund/mitiq/pull/2320/commits/97e309f284e59445af330082b5d33c78ebb1c3fb.